apps/guardian-ui/src/GuardianApi.ts

```diff
@@ -332,4 +351,13 @@ export class GuardianApi
   connectionCode = (): Promise<string> => {
     return this.base.call(AdminRpc.connectionCode);
   };
+
+  config = (connection: string): Promise<ConfigResponse> => {
+    return this.base.call<ConfigResponse>(AdminRpc.config, connection);
+  };
+
+  moduleApiCall = <T>(moduleId: string, rpc: ModuleRpc): Promise<T> => {
+    const method = `${AdminRpc.module}_${moduleId}_${rpc}`;
```

Comment:
`moduleId` is a u16 in fedimint-core/src/api.rs, should this be `number`?

Reply:
fixed